packages/extensions/openapi-to-cadl/test/anomalyDetector/cadl-output/models.cadl

```diff
@@ -495,7 +483,7 @@ the anomaly score of the subsequent point.
   alignPolicy?: AlignPolicy;
 
   @doc("Model status. One of CREATED, RUNNING, READY, and FAILED.")
-  status?: ModelStatus;
+  status?: modelStatus;
```

Comment:
This seems wrong? TypeSpec model names should be PascalCase, so this should be `ModelStatus`.

Reply:
So this is coming from removing `prenamer` and it is taking the name straight from Swagger. Prenamer was generating a few `AutoGenerated` names and also some weird Enum definitions, for example, it was creating some aliases that didn't make sense.

For example, it was generating
![image](https://user-images.githubusercontent.com/21109913/221058623-0e3fc521-a7f5-4b17-8d86-bb2879c5d058.png)

Where we'd expect  this, which is what swagger has:
![image](https://user-images.githubusercontent.com/21109913/221058668-db628700-1f6e-4f0e-8391-bb30bf908f46.png)

Reply:
Added pretransform to fix this casing